src/Provider/HostIp/HostIp.php

```diff
@@ -76,42 +38,15 @@ public function getName(): string
      *
      * @return Collection
      */
-    private function executeQuery(string $url): AddressCollection
+    protected function executeQuery(string $url): AddressCollection
     {
         $content = $this->getUrlContents($url);
-        $data = json_decode($content, true);
+        $data = \json_decode($content, true);
```

Comment:
The `\` is not necessary.
```suggestion
        $data = json_decode($content, true);
```

Reply:
replaced with `use`